pages/dkp/kaptain/2.1.0/configuration/external-dex/index.md

```diff
@@ -95,6 +95,8 @@ ingress:
   oidcProviderBase64CaBundle: <OIDC provider CA bundle>
 ```
 
+If you want to use an Active Directory server to authenticate users to access Kaptain, add the corresponding variable as shown in the [Authenticate users using Active Directory](../../user-management/active-directory/) documentation.
```

Comment:
this might be how I perceive this, but the way it's worded it seems as if in order to auth via AD we have to add the variable on top of doing what's included in this page (whilst what's included in this page is not needed for that).
if you see my point you could rephrase to something like "If you want to use an Active Directory server to authenticate users to access Kaptain, follow the guidance on this page (+link)"

but to be fair this might just be me. 
I can definitely approve without this being updated